examples/index.html

```diff
@@ -55,22 +55,30 @@ <h4><b>By entering co-ordinates</b></h4>
 
       var BlurredLocation = new BlurredLocation(options);
 
+      /* 
+        zoom_level <= 5              :  7 Markers
+        zoom_level between 6 and 7   :  5 Markers 
+        zoom_level between 8 and 10  :  3 Markers 
+        zoom_level >= 11             :  2 Markers
+      */
+
+      locations = [[23.1 ,      77.1],
```

Comment:
Do we need to provide a "precision" value too, optionally? I worry about relying on the stored precision of JavaScript values. Or at least we should document the common "gotchas".

Reply:
Currently calculating precision using : 
`afterDecimal = lat.toString().split(".")[1] ;`
I think we should definitely document these 😄 !